src/App.js

```diff
@@ -1,62 +1,84 @@
-import {Navbar} from "./components/navbar/Navbar";
 import {LandingPage} from "./components/LandingPage/LandingPage";
-import {BrowserRouter as Router, Route, Switch} from "react-router-dom";
+import {BrowserRouter as Router, Link, Route, Switch} from "react-router-dom";
 import {Meeting} from "./components/meeting/Meeting";
 import './App.css'
 import {ApplyForm} from "./components/offers/applyForm/ApplyForm";
 import {AllOffersView} from "./components/offers/list/AllOffersView";
 import {OfferForm} from "./components/offers/createForm/OfferForm";
+import {colors} from "./utils/colors";
+import {constants} from "./utils/constants";
+import {AppBar, Button, Toolbar, Typography} from "@material-ui/core";
 
 
 function App() {
   return (
-    <div>
-      <Navbar />
-      <div>
-          <Router>
-              <Switch>
-
-                  <Route exact path="/">
-                      <div style={{marginTop: "2em"}}>
-                          <LandingPage />
-                          <a href='https://www.freepik.com/vectors/people' style={{display:"none"}}>People vector created by pikisuperstar - www.freepik.com</a>
-                      </div>
-                  </Route>
-
-                  <Route exact path="/interview/:interviewId/:companyId">
+      <Router>
+
+          { /* NAVBAR */}
+          <AppBar position="sticky" style={{ background: colors.main, height: `${constants.navbar_height}` }}>
```

Comment:
To że nie mamy navbara w osobnym komponencie, wynika bardziej że robimy to na jutro, czy też z tego że ciężko to zrobić?

Reply:
Na razie z tego, że robimy to na jutro. Ogólnie mieliśmy navbara w osobnym, ale wtedy linki nie działały - możliwe, że dlatego, że był to osobny router, ale to jeszcze sprawdzę po wtorku, tylko muszę o tym pamiętać XD

Reply:
Zrobiłem taska na jirze by nie zapomnieć